netbox_custom_objects/models.py

```diff
@@ -164,9 +164,15 @@ class CustomObjectType(PrimaryModel):
     _through_model_cache = (
         {}
     )  # Now stores {custom_object_type_id: {through_model_name: through_model}}
-    name = models.CharField(max_length=100, unique=True)
+    name = models.CharField(
+        max_length=100,
+        unique=True,
+        help_text=_("Internal lowercased object name, e.g. \"vendor_policy\""),
+    )
     schema = models.JSONField(blank=True, default=dict)
+    verbose_name = models.CharField(max_length=100, blank=True)
     verbose_name_plural = models.CharField(max_length=100, blank=True)
+    slug = models.SlugField(max_length=100, null=True, unique=True, db_index=True)
```

Comment:
I think we can add the help_text here?  same comment above, do we want to enforce lowercase?

Also - do we want this auto-generated off of name (can change but gives a default)?